CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+
+### 3.3.0 - [#51](https://github.com/openfisca/country-template/pull/51)
```

Comment:
Why not `3.2.4` ?

Reply:
Some argue that "making a country package compatible with a new major version of core" is more a "new feature" than a "bug fix". I have no strong opinion on the matter, but several times I've been told to use minors 🙂